examples/app-dir-experiments/app/ssr/ApolloWrapper.tsx

```diff
@@ -29,9 +26,14 @@ function makeClient() {
             new SSRMultipartLink({
               stripDefer: true,
             }),
-            httpLink,
+            new SchemaLink({
+              schema,
```

Comment:
what if we need to create a `context` here with `req` to determine user's authentication state in graphql resolvers?